spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,6 +32,55 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile5.0-RC1]]
+=== MicroProfile 5.0 Release Candidate 1 (Q32021)
+// MicroProfile 5.0-RC1 is the 16th platform release for the
```

Comment:
In 4.1, we decided to drop the number of platform releases we have done. So this "16th" notion can be removed

Reply:
line is commented, anyway I'm removing it!!